src/Mapbender/CoreBundle/Resources/public/mapbender.element.layertree.tree.js

```diff
@@ -276,10 +213,14 @@
                     cursor: "move",
                     update: function(event, ui) {
                         var $elm = $(ui.item);
-                        if ($elm.hasClass('themeContainer')) {
-                            self._sortTheme($elm);
+                        var type = $elm.attr('data-type');
+                        if (type === 'theme' || type === 'root') {
```

Comment:
switch case would be easíer to extend in the future

Reply:
[Wish granted](https://github.com/mapbender/mapbender/pull/852/commits/58247f30d7e8021ceb566c21d0ef5d6ac5ad92df)